deployment/README.md

```diff
@@ -0,0 +1,43 @@
+# Trillian supported deployments
+
+This directory contains the supported deployment types for Trillian. Currently,
+this includes:
+
+- [Kubernetes on GCP using Cloud
+  Spanner](#kubernetes-on-gcp-using-cloud-spanner)
+
+Further deployment types (community-supported) can be found in the
+examples/deployment directory.
+
+# Kubernetes on GCP using Cloud Spanner
+
+## Architecture
+
+The infrastructure is created using Terraform. It consists of a Cloud Spanner
+database, a Trillian-specific etcd cluster, the Trillian logserver and
+logsigner. A Workload Identity is set up by Terraform to give the services access
+to Cloud Spanner. The container images are based on v1.3.3.
+
+## Prerequisites
+1. [Terraform](https://www.terraform.io/), [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/),
+   and [gcloud](https://cloud.google.com/sdk/gcloud/) are installed.
+1. You have a Google account with billing configured.
+1. Terraform should have access to the credentials to manage your Cloud Project. Follow the
+   instructions from the [Terraform Google Provider documentation](https://www.terraform.io/docs/providers/google/guides/provider_reference.html) to set this up.
+
+## Installation
+1. Create a new project and copy its [Project
+   ID](https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects).
+1. Create the infrastructure: `terraform init && terraform apply -var="gcp_project=PROJECT_ID"`.
```

Comment:
When executing this step, I get the following error:
```
Error: google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.

  on main.tf line 10, in provider "google":
  10: provider "google" {
```
It's not completely obvious which instructions, if any, I should follow on the linked page.

Reply:
I found that running ` gcloud auth application-default login` fixed this. The complete instructions are here: https://www.terraform.io/docs/providers/google/guides/provider_reference.html#full-reference. I think it'd be helpful to add this as a step above.

Reply:
Ah I see this was linked to from the prerequisites section. I'd add that #full-reference anchor to that link, since that takes the reader straight to the relevant instructions.